chart/values.yaml

```
@@ -563,6 +563,24 @@ scheduler:
     timeoutSeconds: 20
     failureThreshold: 5
     periodSeconds: 60
+    exec:
+      command:
+        - sh
+        - -c
+        - |
+          CONNECTION_CHECK_MAX_COUNT=0 /entrypoint python -Wignore -c "
```

```suggestion
          CONNECTION_CHECK_MAX_COUNT=0 exec /entrypoint python -Wignore -c "
```

This is where the exec should be, and I think we want to keep it so `timeoutSeconds` can actually function.

---

Yep.